RealmSwift/Results.swift

```diff
@@ -284,3 +284,13 @@ extension Results: CollectionType {
     /// endIndex is not a valid argument to subscript, and is always reachable from startIndex by zero or more applications of successor().
     public var endIndex: Int { return count }
 }
+
+// MARK: Variadic Arguments Support
+
+extension Results: CVarArgType {
+    /// Transform `self` into a series of machine words that can be
+    /// appropriately interpreted by C varargs
+    public func encode() -> [Word] {
+        return (map(self, { $0 }) as NSArray).encode()
```

Comment:
Because that would just be way too simple! /jk, will do. it hadn't occurred to me.